proj/src/main/webapp/script.js

```diff
@@ -233,6 +233,9 @@ function init() {
     
     auth.onAuthStateChanged(async firebaseUser => {
         if(firebaseUser){
+
+            clickWithEnterKey();
+
```

Comment:
Now that there is more on the page than just the chat, you might need to revisit how this listener is done. If they are typing their bio and press enter, shouldn't it submit the bio not the chat? Maybe find which thing is in focus in `clickWithEnterKey` before submitting the chat?

Reply:
the function listens for the enter key within the text box; if your are pressing enter anywhere else on the page, the message will not be sent